front/src/utils/aroundMe/aroundMe.util.test.ts

```diff
@@ -0,0 +1,59 @@
+import type { LatLng, Region } from "react-native-maps";
+
+import { AroundMeConstants } from "../../constants";
+import { getLatLngPoint } from "./aroundMe.util";
+
+describe("AroundMeUtils", () => {
+  describe("getLatLngPoint", () => {
+    const regionNantes: Region = {
+      latitude: 47.223324097274556,
+      latitudeDelta: 0.13430321917147126,
+      longitude: -1.538015529513359,
+      longitudeDelta: 0.13483263552188873,
+    };
+
+    it("Center point of region", () => {
```

Comment:
peut-être renommer en `should return center point of a given region` ? :) et renommer les autres "it" de la même façon ?